docker-compose.build.yml

```diff
@@ -66,7 +66,7 @@ services:
         - RUST_BUILD=${RUST_BUILD:-debug}
 
   model-plugin-deployer:
-    image: ${DOCKER_REGISTRY:-docker.io}/grapl/model-plugin-deployer:${TAG:-latest}
+    image: grapl/model-plugin-deployer:${TAG:-latest}
```

Comment:
I notice that the `node-identifier-retry` entry above is the only other one that still has this `DOCKER_REGISTRY` variable; does that also need to be removed here?

Reply:
Yep, adding that now. Nice catch